src/htm/algorithms/Connections.hpp

```diff
@@ -649,24 +714,28 @@ class Connections : public Serializable
   std::vector<SynapseData> synapses_;
   std::vector<Synapse>     destroyedSynapses_;
   Permanence               connectedThreshold_; //TODO make const
+  UInt32 iteration_ = 0;
 
   // Extra bookkeeping for faster computing of segment activity.
   std::unordered_map<CellIdx, std::vector<Synapse>> potentialSynapsesForPresynapticCell_;
   std::unordered_map<CellIdx, std::vector<Synapse>> connectedSynapsesForPresynapticCell_;
   std::map<CellIdx, std::vector<Segment>> potentialSegmentsForPresynapticCell_;
   std::map<CellIdx, std::vector<Segment>> connectedSegmentsForPresynapticCell_;
 
-  std::vector<Segment> segmentOrdinals_;
-  std::vector<Synapse> synapseOrdinals_;
```

Comment:
removed the ordinals vectors, this is now stored in `Sement/SynapseData` with no significant overhead, so just the code is clearer.